### PR TITLE
Miscellaneous Raven Crispification

### PR DESCRIPTION
**Pull request by mikeday0**

### CODE REVIEW

File: src/heretic/mn_menu.c [hunk 1]
```diff
@@ -454,6 +456,13 @@ static multiitem_t multiitem_difficulties[NUM_SKILLS] =
     {SKILL_HNTR, "YELLOWBELLIES"},
 };
 
+static multiitem_t multiitem_sndchannels[3] =
```
Comment (turol) [inline src/heretic/mn_menu.c:477]: Should be `const`

File: src/hexen/mn_menu.c [hunk 2]
```diff
@@ -444,6 +446,13 @@ static multiitem_t multiitem_difficulties[NUM_SKILLS] =
     {SKILL_HNTR, "EASY"},
 };
 
+static multiitem_t multiitem_sndchannels[3] =
```
Comment (turol) [inline src/hexen/mn_menu.c:467]: Likewise `const`

**Comment by mikeday0** [comment]
Calling this one done. I would have liked to add the slime trail fix to Hexen, but it's beyond a simple copy-and-paste due to the polyobjects. I'll handle that one in its own PR.

File: src/hexen/h2_main.c [hunk 3]
```diff
@@ -1124,7 +1125,28 @@ static void CrispyDrawStats (void)
     right_widget_h = 0;
     right_widget_x = coord_x + WIDESCREENDELTA;
 
-    if (player->cheats & CF_SHOWFPS)
+    if (crispy->playercoords == WIDGETS_ALWAYS || (automapactive && crispy->playercoords == WIDGETS_AUTOMAP))
```
Comment (ceski-1) [inline src/hexen/h2_main.c:1131]: I think we all used the `WIDGETS_ALWAYS` condition by accident, oops. Should be this for each game:
```c
if (automapactive && crispy->playercoords == WIDGETS_AUTOMAP)
```

Comment (fabiangreffrath) [reply src/hexen/h2_main.c:1131]: Why do you mean?

Comment (ceski-1) [reply src/hexen/h2_main.c:1131]: I can set `crispy_playercoords 2` in the cfg which is a cheat according to this: https://github.com/fabiangreffrath/crispy-doom/wiki/Crispness#navigational

Comment (fabiangreffrath) [reply src/hexen/h2_main.c:1131]: It shouldn't be exposed in the menu, but it should still be possible to cheat it on the screen by editing the config file.

File: src/heretic/mn_menu.c [hunk 4]
```diff
@@ -2542,10 +2600,12 @@ static void DrawCrispnessItem(boolean item, int x, int y)
     MN_DrTextA(item ? "ON" : "OFF", x, y);
 }
 
-static void DrawCrispnessMultiItem(int item, int x, int y, const multiitem_t *multi)
+static void DrawCrispnessMultiItem(int item, int x, int y, const multiitem_t *multi,
+        boolean cond)
 {
-    dp_translation = item ? cr[CR_GREEN] : cr[CR_DARK];
-    MN_DrTextA(multi[item].name, x, y);
+    dp_translation = cond ? cr[CR_GRAY] :
```
Comment (fabiangreffrath) [inline src/heretic/mn_menu.c:2606]: So, if `cond == true` then the item is grayed out?

Comment (mikeday0) [reply src/heretic/mn_menu.c:2606]: The intent is to have this text be the same color as the preceding menu item name. It  appears that `CR_GRAY` maps to the default color (white) but I should probably use `NULL` here instead. Good catch.

**Review by fabiangreffrath — APPROVED** [review]

**Pull request merged**